data_import/tasks.py

```diff
@@ -174,6 +174,8 @@ def insert_salaries(self, *, s_file_id):
     self.import_utility.insert_job()
     self.import_utility.insert_salary()
 
+    call_command('sync_pgviews')
```

Comment:
nice! pretty simple.

Reply:
Can we have the index creation happen here. it's going to lead to confusion to manually set that.

Reply:
like as a separate line.

Reply:
that's a good idea! only downside is it wouldn't work for initial index creation... i don't think we'd be able to create the index in a separate django migration, either, bc the sync management command waits for migrations to be applied first, so the index creation would fail due to view not yet existing... i will open an issue for this (and the other improvements you pointed out).